src/OrbitGl/CallTreeView.cpp

```diff
@@ -197,8 +190,9 @@ std::unique_ptr<CallTreeView> CallTreeView::CreateTopDownViewFromPostProcessedSa
        post_processed_sampling_data.GetSortedThreadSampleData()) {
     const uint32_t tid = thread_sample_data->thread_id;
 
-    for (const auto& [callstack_id, events] : thread_sample_data->sampled_callstack_id_to_events) {
-      uint64_t sample_count = events.size();
+    for (const auto& [callstack_id, callstack_events] :
+         thread_sample_data->sampled_callstack_id_to_events) {
+      uint64_t sample_count = callstack_events.size();
```

Comment:
Nit/Personal preference: as this variable is now only used once, I would either move it closer or even inline it. Same above.

Reply:
I moved it closer to usage (rather, I moved `resolved_callstack` down), but I left it as it's used twice actually.